python/cuml/benchmark/algorithms.py

```diff
@@ -119,6 +126,181 @@ def run_cuml(self, data, **override_args):
         return cuml_obj
 
 
+class AlgorithmFIL:
+    """
+    Wraps a cuML algorithm and other tree inference related algorithms 
+    (scikit-learn, xgboost-cpu, xgboost-gpu, treelite).
+    Provides mechanisms to run each version with default arguments.
+    If no CPU-based version of the algorithm is available, pass None for the
+    cpu_class when instantiating
+    """
+
+    def __init__(
+        self,
+        xgb_class,
+        skl_class,
+        tl_class, 
+        cuml_class,
+        shared_args,
+        cuml_args={},
+        cpu_args={},
+        name=None,
+        data_prep_hook=None,
+        accuracy_function=None,
+    ):
+        """
+        Parameters
+        ----------
+        xgb_class : class
+           Class for XGBoost version of algorithm. Set to None if not available.
+        skl_class : class
+           Class for sklearn version of algorithm. Set to None if not available.
+        tl_class : class
+           Class for treelite version of algorithm. Set to None if not available.           
+        cuml_class : class
+           Class for cuML algorithm
+        shared_args : dict
+           Arguments passed to both implementations's initializer
+        cuml_args : dict
+           Arguments *only* passed to cuml's initializer
+        cpu_args dict
+           Arguments *only* passed to sklearn's initializer
+        data_prep_hook : function (data -> data)
+           Optional function to run on input data before passing to fit
+        accuracy_function : function (y_test, y_pred)
+           Function that returns a scalar representing accuracy
+        """
+
+        XGOBOOST_MODEL_PATH = './'
```

Comment:
Could you rename the variable to `XGBOOST_MODEL_PATH`?

Reply:
Yeah, I will do that tomorrow. Any suggestions for the name of this variable